omero/sysadmins/unix/install-web/install-nginx.txt

```diff
@@ -73,6 +73,12 @@ will depend on your system, please refer to your web server's manual.
 See :ref:`customizing_your_omero_web_installation`
 for additional customization options.
 
+Set the following:
```

Comment:
you can put `::` after 'following' here so you only need one blank line before '$bin/omero...'